infrastructure/main.tf

```diff
@@ -45,7 +45,7 @@ module "submit-your-appeal-frontend" {
 
 module "redis-cache" {
   source      = "git@github.com:contino/moj-module-redis?ref=master"
-  product     = "${var.env != "preview" ? "${var.product}-redis" : "sscs-tribunals-redis-preview"}"
```

Comment:
Should this have changed don't see how it relates to this story?

Reply:
That is a fix? for the current build issue